src/odemis/driver/test/technolution_test.py

```diff
@@ -0,0 +1,679 @@
+import logging
+import unittest
+
+import numpy
+from odemis import model
+from src.openapi_server.models.mega_field_meta_data import MegaFieldMetaData
+from src.openapi_server.models.cell_parameters import CellParameters as CellAcqParameters
+from src.openapi_server.models.field_meta_data import FieldMetaData
+from datetime import datetime
+import time
+
+from odemis.driver.technolution import AcquisitionServer, ASMDataFlow, MirrorDescanner
+from odemis.driver import technolution
+
+URL = "http://localhost:8080/v1"
```

Comment:
The entire test file can only be runnen if the simulator is started (otherwise the objects cannot be created).

Reply:
With TEST_NOHW you have exactly this behavior. All the tests are skipped if HW not available. 

`# Export TEST_NOHW = 1 to prevent using the real hardware

TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing`